cli/version/version.go

```diff
@@ -41,16 +101,39 @@ func BuildVersionString() string {
 		}
 	}
 
-	versionStr := fmt.Sprintf("v%s", version)
+	versionStr := fmt.Sprintf("Version:\t%s", version)
 	versionParts = append(versionParts, versionStr)
 
-	osArchStr := fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH)
+	osArchStr := fmt.Sprintf("OS/Arch:\t%s/%s", runtime.GOOS, runtime.GOARCH)
```

Comment:
It can be helpful to use `text/template` here. Just create a map with the keys you need and use it with the template.